src/client/java/dev/creesch/WebchatClient.java

```diff
@@ -57,6 +68,55 @@ private void handleMessage(Text message, MinecraftClient client) {
         webInterface.broadcastMessage(jsonChatMessage);
```

Comment:
Interesting that this compiles as `jsonChatMessage` doesn't exist here. Just realized this as I am taking a bit of inspiration from this while creating the database stuff and was looking at how you did this. 

No worries btw, I'll deal with potential merge conflicts on my PR since we will be working in the same area.